src/backend/tests/it/utils/pocketic.rs

```diff
@@ -99,7 +101,13 @@ fn init_arg() -> Arg {
         ecdsa_key_name: format!("master_ecdsa_public_key_{}", SUBNET_ID).to_string(),
         allowed_callers: vec![Principal::from_text(CALLER).unwrap()],
         ic_root_key_der: None,
-        supported_credentials: None,
+        supported_credentials: Some(vec![SupportedCredential {
+            ii_canister_id: II_CANISTER_ID.to_string(),
```

Comment:
I want to convert this argument along with `issuer_canister_id` to Principal in another PR before we deploy a version of Oisy with this arguments.